front/app/components/admin/SeatBasedBilling/ChangeSeatModal/index.tsx

```diff
@@ -69,9 +71,10 @@ const ChangeSeatModal = ({
   closeModal,
   userToChangeSeat,
   changeRoles,
-  isChangingToNormalUser,
   returnFocusRef,
+  changingToRoleType,
 }: Props) => {
+  const isChangingToNormalUser = changingToRoleType === 'user';
```

Comment:
I feel this modal is a pretty good example of trying to do way too much in one component. I tried to not make it messier but don't really have the capacity to untangle it properly at the moment.